spec/rubocop/cop/style/string_concatenation_spec.rb

```diff
@@ -201,4 +201,33 @@
       RUBY
     end
   end
+
+  context 'EnforcedStyle = conservative' do
+    let(:cop_config) { { 'EnforcedStyle' => 'conservative' } }
+
+    context 'when first operand is not string literal' do
+      it 'does not register offense' do
+        expect_no_offenses(<<~RUBY)
+          user.name + "!!"
+          user.name + "<" + "user.email" + ">"
```

Comment:
Actually, I would expect an offense here. Not on `user.name + "<"`, but on `"<" + "user.email"` (and `"user.email" + ">"`).

Reply:
Good catch!

Reply:
I disagree.

```ruby
# yes, I know that a variable called user.name is unlikely to be a Pathname
user.name = Pathname.new("user.name")
result = user.name + "<" + "user.email" + ">"
puts "result is #{result}" # result is user.name/</user.email/>
                           # which is in no way equivalent to "user.name<user.email>"
```

Reply:
@tejasbubane -- I think the fix that was called for here was in error.  See my above comment